omnibus/config/software/datadog-agent-env-check.rb

```diff
@@ -1,13 +0,0 @@
-name 'datadog-agent-env-check'
```

Comment:
see comment on the project file for context, logic was moved to the `datadog-agent-integrations` def